packages/notifications/src/notifications-provider.ts

```diff
@@ -5,7 +5,7 @@ import { AnyPromiseFn, withErrorHandling } from '@terra-ui-packages/core-utils';
  */
 export type IgnoreErrorDecider = (title: string, obj?: unknown) => boolean;
 
-export type NotificationType = 'error' | 'warn' | 'info' | 'success' | 'welcome';
+export type NotificationType = 'error' | 'warn' | 'info' | 'success' | 'welcome' | 'custom';
```

Comment:
If anyone has a preferable name, I'm happy to change this.

Reply:
How about "no_icon" or "text_only"?

Reply:
Suggest not using `NotificationType` for this and instead adding an optional `icon` property to `NotificationOptions` that allows overriding the notification icon's default icon.

That way, we get this feature for all notification types and preserve the more semantic meaning of `NotificationType`.